library/jenkins

```diff
@@ -3,7 +3,9 @@
 # maintainer: Carlos Sanchez <csanchez@cloudbees.com> (@carlossg)
 
 latest: git://github.com/jenkinsci/jenkins-ci.org-docker@2fb0684fe6eb11a3c1145d71e0cb50bf2dda8d11
-2.7.4: git://github.com/jenkinsci/jenkins-ci.org-docker@2fb0684fe6eb11a3c1145d71e0cb50bf2dda8d11
+2.7.4: git://github.com/jenkinsci/jenkins-ci.org-docker@6eaa9b15926232310317490a3b9975ef61be763c
```

Comment:
This commit id is the same as it was before (2.19.1 was tagged as 2.7.4). So this is fine.